simtools/db_handler.py

```diff
@@ -182,21 +180,26 @@ def getModelParameters(
 
     def exportFileDB(self, dbName, dest, fileName):
         """
-        Get a file from the DB and write it to disk.
+        Get file from the DB and write to disk.
 
         Parameters
         ----------
         dbName: str
             Name of the DB to search in.
         dest: str or Path
             Location where to write the file to.
-        fileName: str
+        file_name: str
             Name of the file to get.
+        Returns
+        -------
+        GridOut
+            :class:`~gridfs.grid_file.GridOut` or ``None`` if file not found
         """
 
         self._logger.debug(f"Getting {fileName} and writing it to {dest}")
-        file = self._getFileMongoDB(dbName, fileName)
-        self._writeFileFromMongoToDisk(dbName, dest, file)
+        filePathInstance = self._getFileMongoDB(dbName, fileName)
+        self._writeFileFromMongoToDisk(dbName, dest, filePathInstance)
+        return filePathInstance._id
```

Comment:
See above, I don't think it's necessary.

Reply:
I propose we keep this return statement. it will help us to test the function and it does no harm to the code and the user